commands/access/index.js

```diff
@@ -46,7 +46,7 @@ function * run (context, heroku) {
 
     try {
       let admins = yield heroku.get(`/organizations/${orgName}/members`)
-      admins = _.filter(admins, { 'role': 'admin' })
+      admins = admins.filter(a => a.role === 'admin')
```

Comment:
I would probably call the initial response payload "members" instead of "admins" and then when you filter it, do something like:
```const admins = members.filter(member => member.role === 'admin')```